### DISCUSSION TITLE
25 sec (dbus?) timeout on first start of gtk programs

### DISCUSSION THREAD

**Discussion by argrig** [i3/i3#5402] [Q&A]
After update of ubuntu 20.04 I have 25 sec timeout on first start of gtk programs. i3 version is 4.22, but there was the same behavior on 4.17. How to solve the issue?

**Comment by monomono2** [comment]
Hi @argrig 

I had similar problem but on other distribution: [Parabola](https://www.parabola.nu/).

I removed xdg-desktop-portal and xdg-desktop-portal-gtk packages and fixed it.

https://unix.stackexchange.com/questions/748035/i3wm-manjaro-gnome-applications-appear-to-head-dbus-timeout

greetings
